pkg/config/loader.go

```diff
@@ -92,8 +92,17 @@ func ParseWinPDTargets(log logr.Logger, vpcCniConfigMap *v1.ConfigMap) (warmIPTa
 	}
 
 	warmIPTargetStr, foundWarmIP := vpcCniConfigMap.Data[WarmIPTarget]
+	if !foundWarmIP {
```

Comment:
nit: can we just add a comment here that we are transitioning these variables to use the `windows-*` name to avoid confusion? And the older variables will eventually be deprecated?

Reply:
Sure, updated this PR.